classes/Renderer.php

```diff
@@ -420,7 +420,10 @@ public static function output($template, $data = array())
 
         # banner text
         $b = new Model\Banner;
-        $data['banner_text'] = $b->get_text();
+        $text = $b->get_text();
+        if ( $text ) {
+            $data['banner_text'] = $text;
+        }
```

Comment:
This is fine, but would it be simpler to leave this as it was and changing the template to `if ($banner_text)` instead of an `isset`? Would it ever be unset?